js/objectsBasics.js

```diff
@@ -0,0 +1,550 @@
+/*
+OBJECTS
+    An object is a collection of properties. A property is an 
+    association between a name (or key) and a value.
+    An object is created using the curly braces {...}.
+    An empty object can be created using the two ways:
+*/
+    // 1.
+    let personalDetails = new Object(); // object constructor syntax
+
+    // 2.
+    let userDetails = {};  // object literal syntax
+
+    console.log( typeof(personalDetails) ) // object
```

Comment:
Please note that while using semicolons is not necessary, adding them at the end of each JavaScript statement 
 will make your code more readable.

Reply:
Seems like I forgot this. I will be adding them going forward.